src/graphics/vulkan/vulkan_font.cpp

```
@@ -102,7 +102,7 @@ namespace moth_graphics::graphics::vulkan {
 
     Font::Font(FT_Face face, int size, SurfaceContext& context)
```

Clang (14.0.6)

[error] constructor does not initialize these fields: m_glyphAtlas, m_lineHeight, m_ascent, m_descent, m_underline, m_codepointToAtlasIndex, m_glyphBearings, m_shaderInfos, m_vkDescriptorSets, m_glyphInfosBuffer

(cppcoreguidelines-pro-type-member-init,-warnings-as-errors)